src/containers/Member.js

```diff
@@ -130,7 +113,7 @@ const MemberPage = ({ releaseBranch = BRANCH.master }) => {
           <Route exact path={routes.memberPage.path}>
             <RoleCardContainer>
               <MemberRoleCard
-                role={memberData.role}
+                roleKey={memberData.role.key}
```

Comment:
Do we need to give default `memberData` value to ensure that `memberData.role` would not be undefined to prevent js error here ?